stages/org.osbuild.kernel-cmdline

```diff
@@ -1,13 +1,35 @@
 #!/usr/bin/python3
 import os
+import platform
 import sys
 
 import osbuild.api
 
+# COMMAND_LINE_SIZE value per arch
+MAX_SIZE_MAP = {
+    # https://github.com/torvalds/linux/blob/master/arch/x86/include/asm/setup.h
+    "x86_64": 2048,
+    # https://github.com/torvalds/linux/blob/master/arch/x86/include/asm/setup.h
+    "i386": 2048,
+    # https://github.com/torvalds/linux/blame/master/arch/arm/include/uapi/asm/setup.h
+    "arm": 1024,
+    # https://github.com/torvalds/linux/blob/master/arch/arm64/include/uapi/asm/setup.h
```

Comment:
I wonder if this should point to the current tag instead of master so that we know when this was looked up (but then a) git blame will have a date b) this probably does not change very often so I'm probably oeverthinking it.

Reply:
I had the same thought, and opted for master for the same reasons you mention (i.e., git blame will give you a good idea of when was this last looked at, and it is probably not going to change any time soon anyway). Then, the link gives you directly the current value.

That said, I am open to change it :)